proto/attestation/endorsement.proto

```diff
@@ -62,6 +62,9 @@ message KernelLayerEndorsements {
   TransparentReleaseEndorsement init_ram_fs = 4;
   TransparentReleaseEndorsement memory_map = 5;
   TransparentReleaseEndorsement acpi = 6;
+
+  // Field is deprecated and kept only for backwards compatibility. Remove ASAP.
```

Comment:
This seems to just repeat the fact that the field is deprecated, which is already obvious. It would be more useful if this were a TODO with a link to a bug for context.

Reply:
And I don't think the GoB move will address that naturally -- we would still run the same presubmit checks in CI.

Reply:
When I last tried it didn't work. But it's hard to relate it to the merge time of the above.

Reply:
Doesn't work. Filed b/329091025